envrc.el

```diff
@@ -135,7 +136,7 @@ One of '(none on error).")
 ;;; Internals
 
 (defun envrc--lighter ()
-  "Return a colourised version of `envrc--status' for use in the mode line."
+  "Return a colorized version of `envrc--status' for use in the mode line."
```

Comment:
Controversial! I personally shamelessly use international english spellings rather than American English, at least inside comments and docstrings.

Reply:
Ok, fixed :-).